test/over_react/component/dom_components_test.dart

```diff
@@ -83,12 +82,12 @@ main() {
       if (expectedTagName == 'missingGlyph') expectedTagName = 'missing-glyph';
       if (expectedTagName.startsWith(new RegExp('svg.'))) expectedTagName = expectedTagName.substring(3);
 
-      test('Dom.$name generates the correct type', () {
-        DomProps builder = domClassMirror.invoke(element, []).reflectee;
+      test('${method.toString()} generates the correct type', () {
```

Comment:
Using `.toString()` inside a string interpolation is redundant:
```suggestion
      test('$method generates the correct type', () {
```

Reply:
🤦‍♂️